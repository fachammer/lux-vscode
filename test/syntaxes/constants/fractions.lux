+1.0
+0.0
-0.123
+0.123
-1.0
-0.1
+1.0
-123.0
