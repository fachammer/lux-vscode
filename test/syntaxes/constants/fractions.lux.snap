>+1.0
#^^^^ source.lux constant.numeric.fraction.lux
>+0.0
#^^^^ source.lux constant.numeric.fraction.lux
>-0.123
#^^^^^^ source.lux constant.numeric.fraction.lux
>+0.123
#^^^^^^ source.lux constant.numeric.fraction.lux
>-1.0
#^^^^ source.lux constant.numeric.fraction.lux
>-0.1
#^^^^ source.lux constant.numeric.fraction.lux
>+1.0
#^^^^ source.lux constant.numeric.fraction.lux
>-123.0
#^^^^^^ source.lux constant.numeric.fraction.lux
>